templates/user/settings/profile.tmpl

```diff
@@ -47,7 +41,7 @@
 				<div class="ui divider"></div>
 				<!-- private block -->
 
-				<div class="field">
+				<div class="field" id="privacy">
```

Comment:
Is it possible to have a 2 to 3 words for the ID to avoid potential conflicts?

https://docs.gitea.com/contributing/guidelines-frontend#gitea-specific-guidelines

> 2. HTML ids and classes should use kebab-case, it's preferred to contain 2-3 feature related keywords.